nav2_msgs/srv/RemoveShapes.srv

```diff
@@ -0,0 +1,5 @@
+# Remove vector objects from map
+
+unique_identifier_msgs/UUID[] uuids
```

Comment:
Have a bool for `all_objects` so you don't specify uuid, just removes them all

Reply:
[Added](https://github.com/ros-planning/navigation2/pull/3930/commits/d6a831d90d1440436538a61c771f017bddb59422)